### PR TITLE
Several feed issues

### PR DESCRIPTION
**Pull request by javierarce**
This PR fixes several issues in the user feed: 

- Name is not aligned with the beginning of the list.
- Texts on left column icons are displaced 1 or 2 pxs down.
- Space between likes text and icon is wrong (comparing to the other icons + texts)
- Transparent button that could be clicked.
- Increased the number of items in the user feed (from 4 to 8).

/cc @xavijam

### CODE REVIEW

**Comment by xavijam** [comment]
ded06?

File: lib/assets/javascripts/cartodb/user_feed/view.js [hunk 1]
```diff
@@ -122,9 +122,9 @@ module.exports = cdb.core.View.extend({
 
   _onChangeShowMore: function() {
     if (this.model.get('show_more')) {
-      this.$('.js-more').removeClass('is-transparent');
+      this.$('.js-more').removeClass('is-hidden');
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb/user_feed/view.js]: It could be replaced by a ```.toggleClass``` function.

**Comment by xavijam** [comment]
Apart from the comment :+1:.

**Comment by xavijam** [comment]
- [ ] updated or created icon is too close to the text.

screen shot 2015-09-22 at 12 37 52

**Comment by xavijam** [comment]
Apart from the icon+text separation :+1:

**Comment by javierarce** [comment]
Ok, I'll fix that in this issue: https://github.com/CartoDB/cartodb/issues/5671 so I can release the rest of the fixes here.

**Comment by xavijam** [comment]
:+1:

**Pull request merged**